framework/PortalFrame/PortalFrame.bbj

```diff
@@ -70,10 +72,29 @@ class public PortalFrame
             REM btn!.setToolTip(menuitem!.getToolTip())
             REM btn!.setCallback(BBjAPI.ON_BUTTON_PUSH,#this!,"onMenuItemClick")
         wend
+
+        footerMenuItem! = footerMenu!.addMenuItem(footerMenu!.getRoot(),498,"Switch Theme","Switch application theme","WebKit/demo/assets/home.svg")
```

Comment:
is it correct image? `WebKit/demo/assets/home.svg`?

Reply:
No, it's not the correct image. Like I mentioned in the PR's description - 

> Icons are not set properly, waiting for the new icon setting workflow to be finished.

These are example placeholder icon values, will be replaced in a later PR.